kaldb/pom.xml

```diff
@@ -438,6 +439,12 @@
             <version>5.2.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Maybe add a comment here why we have this specific version pinned, and then add a comment to the Mockito version that we can remove Bytebuddy probably after the next Mockito version bump.